Procedural Dungeon Generator

```diff
@@ -38,10 +52,45 @@ not so close as to be unpassable.
 Add Details
 ==============================
       
-        
+        Now the map needs smaller details to help flesh it out. This is when you add in all
+of the objects that will be scattered about the map. Place tables and chairs in rooms that 
+require them. Create specific items that fit the area they're in, such as dangerous chemicals 
+in a science lab or bunks in a military camp. Computer or electrical panels are also very 
+important. They allow the players to interface with the map in a unique way, and in some 
+cases can be pivotal to the story you are trying to play. When building a map, it's usually 
+a good idea to keep it fairly realistic, but the most important thing to remember is that the 
+players must have fun and that the game be interesting. 
```

Comment:
You are absolutely right about this! Fun and/or interesting is the most important thing! :+1: